apps/cqrs-func/local.settings.json.example

```diff
@@ -2,7 +2,6 @@
   "IsEncrypted": false,
   "Values": {
     "FUNCTIONS_WORKER_RUNTIME": "node",
-    "AzureWebJobsStorage": "{AzureWebJobsStorage}",
```

Comment:
this setting is mandatory (it should point to the internal storage used by the function), but should not referenced anywhere in the application code